src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/monitoring/loki/main.tf

```diff
@@ -96,6 +96,22 @@ resource "helm_release" "grafana-promtail" {
   values = concat([
     file("${path.module}/values_promtail.yaml"),
     jsonencode({
+      tolerations = [
+        {
+          key      = "node-role.kubernetes.io/master"
+          operator = "Exists"
+          effect   = "NoSchedule"
+        },
+        {
+          key      = "node-role.kubernetes.io/control-plane"
+          operator = "Exists"
+          effect   = "NoSchedule"
+        },
+        {
+          operator = "Exists"
```

Comment:
runs promtail on all nodes, even those with NoSchedule taints.  Doesn't run on nodes with NoExecute taints.  This is what the nebari-prometheus-node-exporter daemonset does so I copied it here.  Promtail is what exports logs from the node so we still want it to run on the user and worker nodes.